tests/testthat/setup.R

```diff
@@ -13,7 +13,7 @@ md_df <- data.frame(
   Duration = 160
 )
 
-data_df <- data.frame(
+data_df <- data.table::data.table(
   Concentration = rep(c(0, 0, 1, 3), each = 16), 
   ReadoutValue = runif(n, 1000, 5000),
   BackgroundValue = runif(n, 0, 1),
```

Comment:
@darsoo  both - `cell_lines` and `drugs` are data.frame so there is still a mix in the tests. Is there parallel PR with change data.frame into data.table for gDRtestData? Or it is  intentional?
```
cell_lines <- gDRtestData::create_synthetic_cell_lines()
drugs <- gDRtestData::create_synthetic_drugs()
```

Reply:
It is